release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2679,15 +2689,25 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 |Technology Preview
-// Needs to move to GA after Nov 15, 2023
+
 |Hosted control planes for {product-title} on bare metal
 |Technology Preview
 |Technology Preview
-|Technology Preview
-// Needs to move to GA after Nov 15, 2023
+|General Availability
+
 |Hosted control planes for {product-title} on {VirtProductName}
 |Not Available
 |Technology Preview
+|General Availability
+
+|Hosted control planes for {product-title} on {ibmpowerProductName}
+|Not Available
+|Not Available
+|Technology Preview
+
+|Hosted control planes for {product-title} on {ibmzProductName}
```

Comment:
This attribute also is not rendering in the preview.

Reply:
I think the attributes file was recently updated. I'll double-check and replace the ibmzProductName attribute with whatever the new attribute is. Thanks!

Reply:
@bburt-rh The attributes are fixed now. Thanks again!

Reply:
Great. Thanks, @lahinson! LGTM now. Giving it the good-to-merge label